packages/svg-icons/src-cs/Telerik.SvgIcons/Telerik.SvgIcons.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
```

Comment:
I suppose that the real package metadata will be added on a later stage, right?

Reply:
Added basic metadata to both packages. We can extend it further if needed at a later time.